### DISCUSSION TITLE
Sorting by date added (or scrape-on-upload option)?

### DISCUSSION THREAD

**Discussion by SpectralPrincess** [prosthetichead/GarlicPress#90] [Ideas]
I've been using this for awhile and absolutely loving it, but the one issue I have been having is that when I'm adding a fistful of games to a folder that already has a bunch in it, it would be lovely to not have to pick through the alphabetical game list to find the ones I added to scrape the art for (since I don't need to re-scrape the art for that entire console). Is there any way we could be able to sort the games list by the date they were added? If not, would it be possible for an option to scrape the artwork for games when they're added to the card, so I don't have to seek them out after adding? Thanks so much!!

**Comment by 00anwi** [comment]
I've added this in my pull request #96.

https://github.com/prosthetichead/GarlicPress/assets/8648447/3531f1ab-f739-44fa-80fd-e60f7aa3acab

**Comment by SpectralPrincess** [comment]
Yooooo this is huge thank you so much! It's hugely appreciated, this is a great app made even better with this update! :)
